packages/react-cosmos-component-playground/src/components/FixtureList/index.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from 'react';
+import { objectOf, arrayOf, shape, string, func, bool } from 'prop-types';
+import { match } from 'fuzzaldrin-plus';
+import classNames from 'classnames';
+import { uri } from 'react-querystring-router';
+import { FolderIcon, SearchIcon } from '../SvgIcon';
+import styles from './index.less';
+
+const KEY_S = 83;
+const KEY_ESC = 27;
+
+const getFilteredFixtures = (fixtures, searchText) => {
+  if (searchText.length < 2) {
+    return fixtures;
+  }
+
+  const components = Object.keys(fixtures);
+
+  return components.reduce((acc, componentName) => {
+    const matchingFixtures = fixtures[componentName].filter(fixtureName => {
+      const componentAndFixture = componentName + fixtureName;
+      const fixtureAndComponent = fixtureName + componentName;
+
+      // Ensure that the fuzzy search is working in both direction.
+      // component + fixture and fixture + component. That's because the user
+      // can search for fixture name and afterwards for component name and
+      // we want to show the correct result.
+      return (
+        match(componentAndFixture, searchText).length !== 0 ||
+        match(fixtureAndComponent, searchText).length !== 0
```

Comment:
I know this is old code but I vote that we remove matching for `fixtureAndComponent`. 

In time I noticed that this doesn't work that well as it won't prioritize your specific search and the result you want will be further down the list. Also one-way matching (componentAndFixture) is how all major editors work (Sublime, VS Code etc).

I don't know if this change will break a lot of tests but I it's a quick win I think we should do it. WDYT?